newIDE/app/src/EventsSheet/InstructionEditor/InstructionParametersEditor.js

```diff
@@ -206,9 +209,7 @@ const InstructionParametersEditor = React.forwardRef<
           return () => clearTimeout(timeoutId);
         }
       },
-      // We want to run this effect only when the component did mount.
```

Comment:
this comment is not valid anymore?

Reply:
I badly implemented it at the time I think